src/core/utils.js

```diff
@@ -914,3 +914,19 @@ export function msgIdGenerator(namespace, counter = 0) {
     return gen.next().value;
   };
 }
+
+/**
+ * Generates simple Abbreviations from text passed.
+ *
+ * @param {String} text A string like "Permanent Account Number".
+ * @returns {String} abbr Abbreviation like "PAN"
+ */
+
+export function getAbbreviationFromText(text) {
```

Comment:
this should probably live in core/dfn-abbr.js, no? or is it used here?

Reply:
I wasn't sure, just felt like a good Util function. Shifting it back.